library/koji_tag_inheritance.py

```diff
@@ -44,6 +44,10 @@
      description:
        - The priority of this parent for this child. Parents with smaller
          numbers will override parents with bigger numbers.
+       - When defining an inheritance relationship with "state: present", you
+         must specify a priority. When deleting an inheritance relationship
+         with "state: absent", you should not specify a priority. Ansible will
```

Comment:
you *don't have* to specify, probably?

Reply:
I wanted to make it clear that we completely ignore `priority` in this case. Like it's not something that we "optionally" support if you specify it - we're just going to ignore it.